src/ContactList.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import Contact from './Contact';
+
+class ContactList extends Component {
+
+  render() {
+    return (
+      <ul className="contact-list">
+        {this.props.contacts.map(contact => {
+          return (
+            <Contact
+              key={contact._id}
+              avatar={contact.avatar}
+              name={contact.name}
+              occupation={contact.occupation}
+              onClick={this._id}
```

Comment:
Here is where you need to pass the contacts `_id` as a prop.

Also, in `App.js` you passed the function `handleDeleteContact` as the prop `onClick`, you need to pass that down one level again here.

You are saying the `onClick` prop is `this._id`, a couple of things wrong with that.  `this._id` is undefined, and later inside your `Contact` component you are trying to call `props.onClick()` as a function, but its value is `undefined` so that will be an error.

Really what you want to do is pass the `ContactLists` `onClick` prop down to your contact

`onClick={this.props.onClick}`

something like that should do it for you.